apps/builder/app/builder/features/workspace/canvas-tools/text-toolbar.tsx

```diff
@@ -120,6 +120,10 @@ const Toolbar = ({ state, scale }: ToolbarProps) => {
       onClick={(event) => {
         event.stopPropagation();
       }}
+      onFocus={(event) => {
+        // We don't want the logic in the builder to make canvas inert to be triggered
+        event.preventDefault();
```

Comment:
is it not killing tabs and such?

Reply:
I never seen text toolbar was working with keyboard